engines/dotnet/Main.cs

```diff
@@ -354,7 +330,7 @@ static List<Sample> ModelRegexRedux(Config config)
 547899
 
 """;
```

Comment:
This raw string can be indented in the source code without its value containing the indentation spaces. The amount of spaces the final `"""` is indented with is removed from each line.